src/CourseAuthoringRoutes.jsx

```diff
@@ -0,0 +1,28 @@
+import { PageRoute } from '@edx/frontend-platform/react';
+import React from 'react';
+import { Switch, useRouteMatch } from 'react-router';
+import CourseAuthoringPage from './CourseAuthoringPage';
+import { CoursePageResources } from './course-page-resources';
+import ProctoredExamSettings from './proctored-exam-settings/ProctoredExamSettings';
+
+/**
+ * This component and CourseAuthoringPage should maybe be combined once we no longer need to have
+ * CourseAuthoringPage split out for use in LegacyProctoringRoute.  Once that route is removed, we
+ * can move the Header/Footer rendering to this component and likely pull the course detail loading
+ * in as well, and it'd feel a bit better-factored and the roles would feel more clear.
+ */
```

Comment:
Kudos for improving the URL scheme. Could  structure describe the structure within the code comment?

My guess is `https://<authoring_mfe_root>/<course_id>/<page_name>` but I'd rather be sure and not have to look at the `useRouteMatch` docs.